src/main/java/uk/gov/pay/directdebit/mandate/resources/MandateResource.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.pay.directdebit.mandate.resources;
+
+import java.util.Map;
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+import org.slf4j.Logger;
+import uk.gov.pay.directdebit.app.logger.PayLoggerFactory;
+import uk.gov.pay.directdebit.gatewayaccounts.model.GatewayAccount;
+import uk.gov.pay.directdebit.mandate.api.CreateMandateResponse;
+import uk.gov.pay.directdebit.mandate.services.MandateService;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static javax.ws.rs.core.Response.created;
+
+@Path("/")
+public class MandateResource {
+    private static final Logger LOGGER = PayLoggerFactory.getLogger(MandateResource.class);
+    private final MandateService mandateService;
+    @Inject
+    public MandateResource(MandateService mandateService) {
+        this.mandateService = mandateService;
+    }
+    
+    @POST
+    @Path("/v1/api/accounts/{accountId}/mandates")
+    @Consumes(APPLICATION_JSON)
+    @Produces(APPLICATION_JSON)
+    public Response createMandate(@PathParam("accountId") GatewayAccount gatewayAccount, @PathParam("mandateExternalId") String mandateExternalId, Map<String, String> createMandateRequest, @Context UriInfo uriInfo) {
```

Comment:
I think `mandateExternalId` is not used here?

Reply:
👍